docs/Operators.md

```diff
@@ -7630,39 +7826,79 @@ This version of the operator has been available since version 16 of the default
-<dd>Support padding modes for outside grid values: `zeros`(default), `border`, `reflection`. zeros: use 0 for out-of-bound grid locations, border: use border values for out-of-bound grid locations, reflection: use values at locations reflected by the border for out-of-bound grid locations. If index 0 represents the margin pixel, the reflected value at index -1 will be the same as the value at index 1. For location far away from the border, it will keep being reflected until becoming in bound. If pixel location x = -3.5 reflects by border -1 and becomes x' = 1.5, then reflects by border 1 and becomes x'' = 0.5.</dd>
+<dd>This attribute must be one of the strings 'zeros', 'border' and 'reflection'</dd>
+<dt><tt>alpha</tt> : float (default is -0.75)</dt>
+<dd>This attribute is used for bilinear and bicubic interpolation. See above for the formulas involving alpha. The default value matches that of PyTorch.
+<dt><tt>nearest_mode</tt> : string (default is round_to_even)</dt>
+<dd>This attribute specifies the rounding mode used in nearest interpolation and must be one of the strings 'round_prefer_floor' (round .5 down), 'round_prefer_ceil' (round .5 up), 'round_to_even' (round .5 to closest even integer). If the rounding mode is not nearest then this attribute is ignored, though it must still be one of the indicated strings.</dd>
 </dl>
 
+
+
 #### Inputs
 
 <dl>
-<dt><tt>X</tt> (differentiable) : T1</dt>
-<dd>4-D tensor of shape (N, C, H, W), where N is the batch size, C is the numbers of channels, H and W are the height and width of the input data.</dd>
-<dt><tt>grid</tt> (non-differentiable) : T1</dt>
-<dd>Input offset, 4-D tensor of shape (N, H_out, W_out, 2), where H_out and W_out are the height and width of grid and output, Grid specifies the sampling pixel locations normalized by the input spatial dimensions. Therefore, it should have most values in the range of [-1, 1]. If grid has values outside the range of [-1, 1], the corresponding outputs will be handled as defined by padding_mode.</dd>
+<dt><tt>input</tt> (differentiable) : T1</dt>
+<dd>4-D tensor of shape (N, C, H, W), where N is the batch size, C is the numbers of channels, H and W are the height and width of the input data. This is the tensor that GridSample does lookups in to retrieve data that is used to compute the output. It is not allowed for N, C, H or W to be zero.</dd>
+
+<dt><tt>grid</tt> (non-differentiable) : T2</dt>
+<dd>4-D tensor of shape (N, H_out, W_out, 2), where H_out and W_out are the height and width of grid and output. This is the tensor that GridSample uses to determine where to do lookups in input. This parameter is allowed to be integer, but this would typically indicate a mistake as this can only access the input data at its corners. It is not allowed for N, H_out or W_out to be zero.</dd>
 </dl>
 
 #### Outputs
 
 <dl>
-<dt><tt>Y</tt> (differentiable) : T2</dt>
+<dt><tt>output</tt> (differentiable) : T1</dt>
```

Comment:
`Y`, same with `X`.

Reply:
It was described as input/grid/output in this pull request. Please reopen if the contention is that it's not allowed to be called anything other than X/grid/Y.